modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/test/java/com/liferay/dynamic/data/mapping/internal/io/exporter/DDMFormInstanceRecordCSVWriterTest.java

```diff
@@ -85,8 +85,8 @@ public void testWrite() throws Exception {
 		StringBundler sb = new StringBundler(3);
 
 		sb.append("Field 1,Field 2,Field 3,Field 4\n");
-		sb.append("2,esta é uma 'string',false,11.7\n");
-		sb.append("1,esta é uma 'string',,10");
+		sb.append("2,false,esta é uma 'string',11.7\n");
+		sb.append("1,,esta é uma 'string',10");
```

Comment:
Hi @linolaoj,

Can you please explain why these changes were necessary?

Thanks.

Reply:
@natocesarrego, this was reverted since to the way it was before LPS-101288, since the decided behavior change.

Reply:
Hi @linolaoj,

Analyzing your explanation more carefully, I realized that this modification is actually setting the value of `field2` to the `field3` and vice-versa, is it intended? I understand that this change only makes sense if the labels were changed too, like ...,`Field3`, `Field2`, etc. Even though, I didn't see how this change is related to the fix. 😞

Reply:
Hi @natocesarrego, this was the revert of b0ce331adc5018e7e48adf44f0fc4eb1ed5790ac. This test was updated for the behavior from e1f4091d2070b5d8d6a0698fc548fe0018680f21. And I reverted all the changes of LPS-101288 since the expected behavior changed.